.github/actions/post-deployment/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: Post-deployment tests
+description: "Run post-deployment tests for this repo"
+
+inputs:
+  targetEnvironment:
```

Comment:
Could the naming be consistent with the other workflows doing TF stuff like .github/workflows/reusable_internal_repo_build.yaml

i dont mind which way round you rename things, but if we try smush or call between these workflows in future the naming is going to be a mess and harder to unpick then

Reply:
This is an action, since you can't use workflows across repos in that way. Reusable workflows in the internal repo have consistent naming I think, but this is the contents of the actions folder in templates:

<img width="333" alt="image" src="https://github.com/user-attachments/assets/93abcfef-d5eb-43a8-ae90-b3611d6548a0">

This isn't really 'reusable' in the same sense since it's the actual entry point for running the post-deployment tests. Maybe `run-post-deployment-tests` ?

Reply:
I've unified the variable names in the reusable workflows in this repo and IAM, standardising on targetX camel-case matching the variables used in internal.